Components/InputTypeNumber/index.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
+export default function InputTypeNumber({
+  idString,
+  valueString,
+  onChange,
+  labelString,
+  nameString,
+  placeholderString,
+}) {
+  return (
+    <>
+      <NumberInput>
+        <Label htmlFor={idString}>{labelString}</Label>
+        <Input
+          type="number"
+          id={idString}
+          value={valueString}
+          onChange={onChange}
+          name={nameString}
+          placeholder={placeholderString}
+        />
+      </NumberInput>
+    </>
```

Comment:
We don't need the React Fragment if we only return one element. 

```suggestion
      <NumberInput>
        <Label htmlFor={idString}>{labelString}</Label>
        <Input
          type="number"
          id={idString}
          value={valueString}
          onChange={onChange}
          name={nameString}
          placeholder={placeholderString}
        />
      </NumberInput>
```

Reply:
Done.